kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaServiceConfiguration.java

```diff
@@ -113,6 +113,13 @@ public class KafkaServiceConfiguration extends ServiceConfiguration {
     )
     private boolean kafkaEnableMultiTenantMetadata = true;
 
+    @FieldContext(
+        category = CATEGORY_KOP,
+        required = true,
+        doc = "Use to enable/disable Kafka authorization force groupId check."
+    )
+    private boolean kafkaEnableAuthorizationForceGroupIdCheck = false;
```

Comment:
@BewareMyPower Do you have suggestions about this configuration?

Reply:
No, we can improve the docs about this config later.